Chef.PowerShell/PowerShell.cs

```diff
@@ -18,6 +18,7 @@ public partial class PowerShell
         /// Executes a PowerShell script via Windows PowerShell. Requires PowerShell 3.0 or above.
         /// </summary>
         /// <param name="powershellScript">String. Script to execute.</param>
+        /// <param name="timeout">Int. Defaults to -1, indefinite</param>
```

Comment:
that `-1` default was removed.

Reply:
Updating the comment to reflect new reality.